tests/test_trace.py

```diff
@@ -523,10 +503,9 @@ def test_timeout_happyflow_handler_call(wrapped_post):
 @mock.patch('requests.Session.post')
 def test_send_traces_sanity(wrapped_post):
     trace = trace_factory.get_or_create_trace()
-    trace.token = 'a'
```

Comment:
We don't support changing the token on the fly now, because the transport is created on `epsagon.init`